Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "run.py", line 19, in <module>
    from Common.gpu_utils import set_gpu
  File "/home/myk/work/EfficientSegmentation/Common/gpu_utils.py", line 3, in <module>
    import pynvml
ModuleNotFoundError: No module named 'pynvml'
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/anaconda3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/anaconda3/lib/python3.7/site-packages/torch/distributed/launch.py", line 260, in <module>
    main()
  File "/usr/local/anaconda3/lib/python3.7/site-packages/torch/distributed/launch.py", line 256, in main
    cmd=cmd)
subprocess.CalledProcessError: Command '['/usr/local/anaconda3/bin/python', '-u', 'run.py', '--local_rank=7', '--local_rank=0']' returned non-zero exit status 1.
*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
